<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findWindow</class>
 <widget class="QDialog" name="findWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>520</y>
     <width>250</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="find">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>410</y>
     <width>250</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="resultTable">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>701</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>540</y>
     <width>441</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Remark: If the Full ID or All Resource's Information </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="reverseOrder">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>340</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reverse Order</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="sorting">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>340</y>
     <width>201</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="title">
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="author">
      <property name="text">
       <string>Author</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="quantity">
      <property name="text">
       <string>Quantity</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="availability">
      <property name="text">
       <string>Available Resources</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="publicationYear">
      <property name="text">
       <string>Publication Year</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="reservation">
      <property name="text">
       <string>Reservations</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="unsort">
      <property name="text">
       <string>Unsort</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>0</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Script MT</family>
     <pointsize>30</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Display and Find Resources</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>65</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Displaying The Resources:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>330</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sort By :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>560</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>are entered then the result will be UNIQUE.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter The Resource's Information:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>351</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QGroupBox" name="ID">
      <property name="title">
       <string>ID</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QCheckBox" name="isID1">
           <property name="text">
            <string>Type (ID1)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="type">
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QRadioButton" name="book">
              <property name="text">
               <string>Book (0)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="article">
              <property name="text">
               <string>Article (1)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="digitalcontent">
              <property name="text">
               <string>Digital Content (2)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="other">
              <property name="text">
               <string>Other (3)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QCheckBox" name="isID2">
           <property name="text">
            <string>ID2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="id2"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="isResource">
      <property name="text">
       <string>All Resource's Information</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="isPublication">
          <property name="text">
           <string>Publication Year</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="inputPublicationYear"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="isAuthor">
            <property name="text">
             <string>Author</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="showFullAuthor">
            <property name="text">
             <string>Only Find Full Matches</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="inputAuthor"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="isTitle">
            <property name="text">
             <string>Title</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="showFullTitle">
            <property name="text">
             <string>Only Find Full Matches</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="inputTitle"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>470</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Find a Resource Based On Enterred Information</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>580</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Go Back To The Main Window</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
